docs/tutorials/t2ramsey_tutorial.ipynb

```diff
@@ -0,0 +1,416 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "backend_in_ns = T2RamseyBackend(\n",
+    "                    p0={\"a_guess\":[0.5], \"t2ramsey\":[80000], \"f_guess\":[0.00002], \"phi_guess\":[0.0],\n",
+    "                        \"b_guess\": [0.5]},\n",
+    "                    initial_prob_plus=[0.0],\n",
+    "                    readout0to1=[0.02],\n",
+    "                    readout1to0=[0.02],\n",
+    "                    conversion_factor=conversion_factor\n",
+    "                )\n",
+    "exp_in_ns = T2Ramsey(qubit, delays, unit=unit)\n",
+    "exp_in_ns.set_analysis_options(user_p0=None, plot=True)\n",
+    "expdata_in_ns = exp_in_ns.run(backend=backend_in_ns, shots=2000)\n",
+    "print(expdata_in_ns)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "It is possible to add data to an experiment, after the analysis of the first set of data. In the next example we add exp2 to `exp_in_ns` that we showed above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "- t2ramsey_value: 8.11988278439035e-05\n",
+      "- frequency_value: 99019986.28277074\n",
+      "- stderr: 1.1732677035570437e-06\n",
+      "- unit: s\n",
+      "- label: T2Ramsey\n",
+      "- fit: \n",
+      "- reduced_chisq: 1.0009782742997637\n",
+      "- dof: 70\n",
+      "- xrange: [1.0000000000000002e-06, 0.00014900000000000002]\n",
+      "- circuit_unit: ns\n",
+      "- quality: computer_good\n",
+      "- success: True\n"
+     ]
+    },
+    {
+     "data": {
```

Comment:
Right, the connecting line is a bummer here... no luck I guess removing it so far. :-(

Reply:
The connecting line is not supposed to be here anymore, since the T1 tutorial PR has been merged

Reply:
What's the blue line?

Reply:
Indeed the redundant lines disappeared. What did you do in T1 that fixed it?

Reply:
I removed all the lines, it's only scatter now